libr/anal/p/anal_gb.c

```diff
@@ -714,7 +714,10 @@ static bool gb_custom_daa (RAnalEsil *esil) {
 static int gb_anop(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, int len, RAnalOpMask mask) {
 	int ilen = gbOpLength (gb_op[data[0]].type);
 	if (ilen > len) {
-		ilen = 0;
+		op->addr = addr;
```

Comment:
i think this field is set  by the caller, so you dont need to define it here

Reply:
I'm not sure about that. Also if that is the case, why do we pass addr to plugin->op anyway?

Reply:
Good question, we can probably get rid of this argument too

Reply:
But i think it is better to focus on rarch refactoring to do that

Reply:
so far this plugin has set the addr on it's own, and I believe that other plugins do that as well. so not sure if removing that line would be good